sequencer/src/block2.rs

```diff
@@ -647,28 +727,31 @@ mod test {
                 );
 
                 // test ns proof
-                let (ns_payload_flat_from_proof, ns_proof) = block
-                    .namespace_with_proof(&actual_ns_table, ns_idx)
+                let ns_proof2 = block
```

Comment:
Why the variable name `ns_proof_2`?

Reply:
Sorry I forgot to change that. I'll fix it.

Reply:
done in https://github.com/EspressoSystems/espresso-sequencer/pull/1073/commits/5c461b4331ef71db1ffe9362ce5ea2677fb51407